src/app/app.component.html

```diff
@@ -14,5 +55,4 @@
     </div>
   </article>
 </main>
-<prx-footer></prx-footer>
```

Comment:
No longer needed with the global nav. However, removing the footer leaves a large void at the end of the page. Again, if I can log in, I can take a look.